server/plugin/command.go

```diff
@@ -85,6 +88,123 @@ func (p *Plugin) getGithubClient(userInfo *GitHubUserInfo) *github.Client {
 	return p.githubConnect(*userInfo.Token)
 }
 
+func (p *Plugin) getMutedUsernames(userInfo *GitHubUserInfo) []string {
+	mutedUsernameBytes, err := p.API.KVGet(userInfo.UserID + "-muted-users")
+	if err != nil {
+		return nil
+	}
+	mutedUsernames := string(mutedUsernameBytes)
+	var mutedUsers []string
+	if len(mutedUsernames) == 0 {
+		return mutedUsers
+	}
+	mutedUsers = strings.Split(mutedUsernames, ",")
+	return mutedUsers
+}
+
+func (p *Plugin) handleMuteList(args *model.CommandArgs, userInfo *GitHubUserInfo) string {
+	mutedUsernames := p.getMutedUsernames(userInfo)
+	var mutedUsers string
+	for _, user := range mutedUsernames {
+		mutedUsers += fmt.Sprintf("- %v\n", user)
+	}
+	if len(mutedUsers) == 0 {
+		return "You have no muted users"
+	}
+	return "Your muted users:\n" + mutedUsers
+}
+
+func contains(s []string, e string) bool {
+	for _, a := range s {
+		if a == e {
+			return true
+		}
+	}
+	return false
+}
+
+func (p *Plugin) handleMuteAdd(args *model.CommandArgs, username string, userInfo *GitHubUserInfo) string {
+	mutedUsernames := p.getMutedUsernames(userInfo)
+	if contains(mutedUsernames, username) {
+		return username + " is already muted"
+	}
+
+	if strings.Contains(username, ",") {
+		return "Invalid username provided"
+	}
+
+	var mutedUsers string
+	if len(mutedUsernames) > 0 {
+		// , is a character not allowed in github usernames so we can split on them
+		mutedUsers = strings.Join(mutedUsernames, ",") + "," + username
+	} else {
+		mutedUsers = username
+	}
+	if err := p.API.KVSet(userInfo.UserID+"-muted-users", []byte(mutedUsers)); err != nil {
+		return "Error occurred saving list of muted users"
+	}
+	return fmt.Sprintf("`%v`", username) + " is now muted"
```

Comment:
> One question for you... Is the help clear enough? Not having used this yet I wonder if the help info be added should explicitly state that the mute list is muting comment from users on your PR and Issues and it does not include @ mention . This is made clear in the title of this PR but, the help does not give this context. It only explains that you're adding a user to your mute list.

That is a good point. Maybe we can improve the wording here and say: `$USER is now muted. You will not longer receive notifications for comments in your PRs and issues.`